speedtest/server.go

```diff
@@ -254,6 +272,25 @@ func (s *Speedtest) FetchServerListContext(ctx context.Context) (Servers, error)
 		if err = decoder.Decode(&servers); err != nil {
 			return servers, err
 		}
+		var CountryCode = ""
+		if s.config.Location != nil {
+			CountryCode = strings.ToUpper(s.config.Location.CC)
+		} else if len(s.config.CountryCode) > 0 {
+			CountryCode = strings.ToUpper(s.config.CountryCode)
+		} else {
+			defaultConfig, err := FetchUserInfo()
+			if err != nil {
+				return servers, err
+			}
+			CountryCode = defaultConfig.Country
+		}
+		var tmpServers Servers
+		for _, server := range servers {
+			if server.CC == CountryCode {
+				tmpServers = append(tmpServers, server)
+			}
+		}
+		servers = tmpServers
```

Comment:
This is not a good way because too much prefetchs will affect the user experience. (FetchUserInfo will be executed twice)

Reply:
am put the`FetchUserInfo ` here because i dont know how to get from the previous context